ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -46,6 +46,7 @@ StatusAppThreePanelLayout {
 
     Connections {
         target: root.rootStore.chatCommunitySectionModule
+        ignoreUnknownSignals: true
```

Comment:
Do we expect `chatCommunitySectionModule` object to be a different type depending on scenario? If not, then it's not the correct usage (just masking the real issue), see: https://doc.qt.io/qt-5/qml-qtqml-connections.html#ignoreUnknownSignals-prop

What is the root cause of the warning here?

Reply:
@osmaczko I did it cos `chatCommunitySectionModule` can be `null`.

Reply:
I think I can change it to `enabled: !!chatCommunityModule`

Reply:
Then, I believe, `enabled` property should be used instead.
`enabled: root.rootStore.chatCommunitySectionModule != null`

Also according to `target` property docs, it should not produce any warning in that case, but I can get it wrong:
> If set to null, no connection is made and any signal handlers are ignored until the target is not null.

_Please remove `ignoreUnknownSignals` from different places in this PR as well_

Reply:
@iurimatias this comment should be resolved (I should have used "require changes"). @borismelnik could you please create a followup PR?